server/modules/sslComponentsRepository.js

```diff
@@ -28,7 +28,6 @@ module.exports = function SSLComponentsRepository() {
   };
 
   function loadSSLComponentsFromS3(mainCallback) {
-    // TODO(filip): get rid of async
```

Comment:
Let go, there was a ticket but was closed. async is almost gone - it's left only in few dodgy places executed only on production side of things.

As for TODO I find them a good way to document some minor tech debt that I'm coming back to when I have more time, I don't create a ticket for all of these